### DISCUSSION TITLE
How to remove vocal humming from songs?

### DISCUSSION THREAD

**Discussion by m41c0n** [Anjok07/ultimatevocalremovergui#1052] [Q&A]
The application is interpreting vocal humming as an intrument, and it's not removing it...
Is there a model that complete removes the humming?

**Comment by celeste-ex3** [comment]
Here Is You Answer Do as [He Says,](https://github.com/Anjok07/ultimatevocalremovergui/discussions/689#discussioncomment-6644349)
But I Think The (VR) aggressive de-echo model Would Work Best
If It Does Not Then Follow The Above Steps Given By [That Guy](https://github.com/Anjok07/ultimatevocalremovergui/discussions/689#discussioncomment-6644349)

**Reply by m41c0n** [reply]
Thank you so much for this answer!
